include/config.js

```diff
@@ -131,6 +131,18 @@ Configuration.getBaseConfig = function(multisite) {
             globalRoot: 'http://global.localhost:8080'
         },
 
+        //Sets the base template service reprocessing property
+        TemplateService: { useReprocess: false },
+
+        //provides configuration options for localization service
+        "localization": {
+            //enables/disables localization service using the database
+            "db":false,
```

Comment:
some managed environments are not going to allow write access to disk for running web processes.  Setting the default to false could cause errors when attempting to save overrides.  Taking the approach where all overrides are saved in the DB but are synced and held in memory by each process would eliminate the need for this setting and protect our users from disk write errors.